README.md

```diff
@@ -1,56 +1,76 @@
-Gnome Alacritty Toggle
-----------------------
+# Gnome Alacritty Toggle
 
-Simple [gnome-shell](https://wiki.gnome.org/Projects/GnomeShell) (v3.38+) extension to toggle [Alacritty](https://github.com/alacritty/alacritty) window.
+Simple [gnome-shell](https://wiki.gnome.org/Projects/GnomeShell) extension to toggle [Alacritty](https://github.com/alacritty/alacritty) window.
 
 Works under both `Wayland` and `X11`
 
-Hotkey: `Alt+z`
+By default this extension will :
+- Register the `Alt+z` hotkey to toggle Alacritty's shown/hidden state
+- If no terminal is open, it tries to execute `/usr/bin/alacritty`.
+- When opening Gnome's overview (`<Super>` key), Alacritty's window will be hidden.
 
-Installation
-------------
+You can configure this behavior by changing settings through [gsettings](#configuration)
+
+## Requirements
+
+- Gnome `>= 3.38`, `<= 42`
+- Alacritty
+
+## Installation
+
+### Through gnome extensions
 [<img src="https://github.com/andyholmes/gnome-shell-extensions-badge/raw/master/get-it-on-ego.png" width=160px>](https://extensions.gnome.org/extension/3942/toggle-alacritty/)
 
-### or manually
+### Manually
+
+Install the extension
+```bash
+git clone https://github.com/axxapy/gnome-alacritty-toggle ~/.local/share/gnome-shell/extensions/toggle-alacritty@itstime.tech
+```
+
+On Wayland, logout and login so `gnome-extensions` picks up the newly installed extension.  
+On X11, press `Alt+F2`, type `r`, and press enter to reload Gnome.
+
+Then enable the extension :
 ```bash
-$ git clone https://github.com/axxapy/gnome-alacritty-toggle ~/.local/share/gnome-shell/extensions/toggle-alacritty@itstime.tech
-$ gnome-extensions enable toggle-alacritty@itstime.tech
+gnome-extensions enable toggle-alacritty@itstime.tech
 ```
 
-Settings
--------------
+## Configuration
 
 Show alacritty's window on overview screen:
 ```bash
-$ gsettings --schemadir ~/.local/share/gnome-shell/extensions/toggle-alacritty@itstime.tech/schemas set org.gnome.shell.extensions.toggle-alacritty hide-on-overview false
```

Comment:
`$` sign means that command should be executed with user's privileges
`#` sign means that command should be executed with root's privileges
Could you please rebase your PR and get `$` signs back?